DayDatePicker/Classes/DayDatePicker/DayDatePickerView.swift

```diff
@@ -277,45 +281,54 @@ extension DayDatePickerView : UITableViewDataSource, UITableViewDelegate {
         cell.textLabel?.font = textFont
         cell.backgroundColor = UIColor.white
         cell.textLabel?.textColor = textColor
-
+        
         if tableView == dayTableView {
             let date = Date(year: year, month: month, day: dayRange.lowerBound + indexPath.row)
             if let minDate = minDate, date < minDate {
                 cell.textLabel?.textColor = UIColor.lightGray
             }
-
+            
```

Comment:
Can you revert these whitespace changes?

Reply:
I'm reading that probably is some of my git setting. Sorry 😓